Mathlib/RingTheory/Flat/Basic.lean

```diff
@@ -164,7 +167,13 @@ lemma of_linearEquiv [f : Flat R M] (e : N ≃ₗ[R] M) : Flat R N := by
   have h : e.symm.toLinearMap.comp e.toLinearMap = LinearMap.id := by simp
   exact of_retract _ _ _ e.toLinearMap e.symm.toLinearMap h
 
-open DirectSum
+end Flat
+
+namespace Flat
```

Comment:
```suggestion
```
You close the namespace and reopen it. It this is because of some `variable` you can introduce sections.

Reply:
Fixed